superset-frontend/tsconfig.json

```diff
@@ -1,25 +1,16 @@
 {
   "compilerOptions": {
-    "allowJs": true,
-    "allowSyntheticDefaultImports": true,
-    "baseUrl": ".",
-    "composite": true,
-    "declaration": true,
-    "declarationMap": true,
-    "esModuleInterop": false,
-    "forceConsistentCasingInFileNames": true,
-    "importHelpers": false,
-    "jsx": "preserve",
-    "lib": ["dom", "dom.iterable", "esnext"],
-    "module": "esnext",
-    "moduleResolution": "node",
-    "noEmitOnError": true,
+    /* Type Checking */
     "noImplicitAny": true,
     "noImplicitReturns": true,
     "noImplicitThis": true,
     "noUnusedLocals": true,
-    "outDir": "./dist",
-    "pretty": true,
```

Comment:
["pretty"](https://www.typescriptlang.org/tsconfig#pretty) default value is true, remove it.